app/functions/100ms/notificationReminder/index.js

```diff
@@ -0,0 +1,172 @@
+var dotenv = require("dotenv");
+dotenv.config({ path: ".env" });
+const sendPushNotification = require("../../neptune/neptuneCaller");
+const Profile = require("../../../models").Profile;
+
+const moment = require("moment-timezone");
+
+const knex = require("knex")({
+  client: "pg",
+  connection: {
+    host: process.env.DB_HOST,
+    port: process.env.DB_PORT || 5432,
+    user: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_NAME,
+    application_name: process.env.APP_NAME || "nobe-runner",
+  },
+});
+
+const sendTimeBasedReminderNotification = async () => {
+  try {
+    console.log("handle sending notification before 1 hr and before 24 hrs");
+    const tzTime = moment()
+      .utc()
+      .tz("asia/kolkata")
+      .format("YYYY-MM-DD HH:mm:ss");
+
+    //get all the video consultations where buyer has done the payment done
+    const videoConsultation = await knex("video_consultations")
+      .whereIn("request_status", ["buyer_payment_done"])
+      .where("type", "future_date");
+
+    for (i = 0; i < videoConsultation.length; i++) {
+      const consultationRequest = videoConsultation[i];
+
+      var videoConsulationdate = JSON.stringify(
+        consultationRequest.query_date_time
+      ).replaceAll('"', "");
+
+      var convertedConsultationDate = moment(videoConsulationdate).format(
+        "YYYY-MM-DD HH:mm:ss"
+      );
+
+      var timeForConsultationInHours = moment(convertedConsultationDate).diff(
+        moment(tzTime),
+        "hours"
+      );
+
+      // console.log("hours difference", timeForConsultationInHours);
+      var minutesDiff = moment(convertedConsultationDate).diff(
+        moment(tzTime),
+        "minutes"
+      );
+      // console.log("Minutes:", minutesDiff);
+
+      let bankerProfile = await Profile.findOne({
+        where: {
+          uuid: consultationRequest.banker_uuid,
+          type: "fm-seller",
+        },
+      });
+
+      let buyerProfile = await Profile.findOne({
+        where: {
+          uuid: consultationRequest.buyer_uuid,
+          type: "fm-buyer",
+        },
+      });
+
+      // sending notification to banker and buyer before 24 hr of the consultation
+      if (timeForConsultationInHours === +24 && minutesDiff === +1440) {
```

Comment:
@ChaitanyaV98 what is +24 here?

Reply:
actually `timeForConsultationInHours` and `minutesDiff` are of number type, so thats why +1, +60 etc.. which will be considered as number data type, +24 is the number of hours.!